NDB.Covid19/NDB.Covid19.iOS/Views/Settings/SettingsPage2/SettingsPage2ViewController.cs

```diff
@@ -60,17 +58,23 @@ void SetTexts()
             HeaderLabel.SetAttributedText(SettingsPage2ViewModel.SETTINGS_PAGE_2_HEADER);
             ContentText.TextContainerInset = new UIEdgeInsets(0, 20, 20, 40);
             BackButton.AccessibilityLabel = SettingsViewModel.SETTINGS_CHILD_PAGE_ACCESSIBILITY_BACK_BUTTON;
-
+            
             NSMutableAttributedString text = new NSMutableAttributedString();
-            text.Append(ApplyStylingToText(intro, FontType.FontRegular, 1.28, 16, 22, false));
-            text.Append(ApplyStylingToText(par1Title, FontType.FontBold, 1.28, 16, 22, false));
-            text.Append(ApplyStylingToText(par1Content, FontType.FontRegular, 1.28, 16, 22, false));
-            text.Append(ApplyStylingToText(par2Title, FontType.FontBold, 1.28, 16, 22, false));
-            text.Append(ApplyStylingToText(par2Content, FontType.FontRegular, 1.28, 16, 22, false));
-            text.Append(ApplyStylingToText(par3Title, FontType.FontBold, 1.28, 16, 22, false));
-            text.Append(ApplyStylingToText(par3Content, FontType.FontRegular, 1.28, 16, 22, false));
+            text.Append(ApplyStylingToText(intro, FontType.FontRegular));
```

Comment:
not getting it, why did you delete the font sizes?

Reply:
they are the same for each paragraph so i've just made them default